apps/src/templates/teacherDashboard/StatsTable.jsx

```diff
@@ -27,7 +27,7 @@ class StatsTable extends Component {
 
   studentsWithCompletedLevelCount = () => {
     const {section, studentsCompletedLevelCount} = this.props;
-    return section.students.map(student => ({
+    return (section.students || []).map(student => ({
```

Comment:
should `section` be a required proptype? or should we check presence of `section` here before looking at `section.students`? seems like we should do one or the other to clarify whether `section` is required or not.